types/DataCube-Landsat/DataCube-Landsat.py

```diff
@@ -0,0 +1,726 @@
+                                            'bandName': 'coastal_aerosol',
+                                            'bandIndex': 8,
+                                            'wavelengthMin': 435.0,
+                                            'wavelengthMax': 451.0,
+                                            'datasetTag': 'QA'
+                                        },
+                                        {
+                                            'bandName': 'pixel_qa',
+                                            'bandIndex': 9,
+                                            'wavelengthMin': 0.0,
+                                            'wavelengthMax': 255.0,
+                                            'datasetTag': 'QA'
+                                        },
+                                        {
+                                            'bandName': 'radsat_qa',
+                                            'bandIndex': 10,
+                                            'wavelengthMin': 0.0,
+                                            'wavelengthMax': 255.0,
+                                            'datasetTag': 'QA'
+                                        },
+                                        {
+                                            'bandName': 'atmos_opacity',
+                                            'bandIndex': 11,
+                                            'wavelengthMin': 0.0,
+                                            'wavelengthMax': 255.0,
+                                            'datasetTag': 'QA'
+                                        },
+                                        {
+                                            'bandName': 'cloud_qa',
+                                            'bandIndex': 12,
+                                            'wavelengthMin': 0.0,
+                                            'wavelengthMax': 255.0,
+                                            'datasetTag': 'QA'
+                                        }
+                                      ],
+                    #GET THE CORRECT BAND INDEX , MIN MAX WAVELENGTH
+
+                    'fields': [self.dacq_auxField,
+                               self.platform_auxField,
+                               self.instrument_auxField,
+                               self.prodtype_auxField,
+                               self.id_auxField]
+                }
+               ]
+
+# ----- ## ----- ## ----- ## ----- ## ----- ## ----- ## ----- ## ----- ##
+# Utility functions used by the Builder and Crawler classes
+# ----- ## ----- ## ----- ## ----- ## ----- ## ----- ## ----- ## ----- ##
+
+class Utilities():
+
+    def readYaml(self,path):        #to read the yaml file located locally.
+        try:
+            with open(path, 'r') as q:
+                try:
+                    doc = (yaml.load(q))
+                except yaml.YAMLError as exc:
+                    print(exc)
```

Comment:
We shouldnt be printing exceptions - these will unnecessarily pop up as errors for unrelated raster type identification process.

Reply:
this has been taken care of